Easydict/NewApp/View/SettingView/Tabs/View/Shortcut/KeyHolderWrapper.swift

```diff
@@ -64,17 +66,13 @@ extension KeyHolderWrapper {
         func recordViewDidEndRecording(_: RecordView) {}
 
         func recordView(_ recordView: RecordView, didChangeKeyCombo keyCombo: KeyCombo?) {
+            if keyCombo == nil { // clear shortcut
+                Shortcut.shared.updateMenu(type)
```

Comment:
This `if` branch could be omitted since you have the optional binding check for `keyCombo` below.

Reply:
Apologies, this is fine. Please ignore this.